packages/devtools_app/lib/src/timeline/flutter/timeline_processor.dart

```diff
@@ -739,9 +225,17 @@ class FullTimelineProcessor extends TimelineProcessor {
         case TraceEvent.durationCompletePhase:
           _handleDurationCompleteEvent(eventWrapper);
           break;
+        // TODO(kenz): add additional support for flows.
         case TraceEvent.flowStartPhase:
+          if (eventWrapper.event.name.contains('PipelineItem')) {
```

Comment:
PipelineItem is another case we're treating magically. Can we make this pattern more restricted so we have a lower risk of false positives? E.g. what is the range of what the event names look like in practice?

Reply:
for a flutter app, PipelineItem events are flow events that signal frame start and frame end. I made this a const and added a comment to explain.